susemanager/susemanager.changes

```diff
@@ -1,3 +1,6 @@
+- Add Oracle Linux 9 bootstrap repositories for Uyuni
+- Add AlmaLinux 9 bootstrap repositories for Uyuni
+- Add Red Hat Enterprise Linux 9 repositories for Uyuni
```

Comment:
"Red Hat Enterprise Linux 9" ? We do not have RES yet, nor do we provide URLs for native RHEL9.
Is it a good idea to provide it already?